cloudcompose/cluster/aws/cloudcontroller.py

```diff
@@ -73,6 +94,34 @@ def _create_instance_args(self, block_device_map):
             'EbsOptimized': ebs_optimized
         }
 
+    def _create_asg_args(self, block_device_map, cloud_init):
+        asg_name      = self.cluster_name
+        vpc_zones     = self.aws['asg']['subnets']
+        cluster_size  = len(vpc_zones.split(','))
+        redundancy    = self.aws['asg']['redundancy']
```

Comment:
Redundancy should be optional and default to 1. So change this like to self.aws['asg'].get('redundancy', 1)